lib/react_on_rails/server_rendering_pool/ruby_embedded_java_script.rb

```
@@ -231,6 +231,7 @@ def parse_result_and_replay_console_messages(result_string, render_options)
           begin
             result = JSON.parse(result_string)
           rescue JSON::ParserError => e
+            return { html: result_string }
             raise ReactOnRails::JsonParseError.new(parse_error: e, json: result_string)
```

rubocop

[warning] Unreachable code detected.

(Lint/UnreachableCode)